R/html_document.R

```diff
@@ -78,6 +99,54 @@
 #'@param extra_dependencies,... Additional function arguments to pass to the
 #'  base R Markdown HTML output formatter \code{\link{html_document_base}}
 #'@return R Markdown output format to pass to \code{\link{render}}
+#'@section Highlighting:
+#'
+#'  Different highlighting engines can be used with a R Markdown HTML documents:
+#'  highlightjs and Pandoc for all languages they support, \pkg{downlit} only
+#'  for R code.
+#'
+#'  highlightjs engine can only be used with the default template (i.e
+#'  \code{template = "default"}) and it has two styles ("default" and
+#'  "textmate"). When activated, a JS script and a CSS file are inserted into
+#'  the output HTML to make it works. For now, this is the default engine for
+#'  the default template - this could change in the future.
+#'
+#'  Pandoc built-in highlighting engine works with any template, default or
+#'  custom, and style can be chosen among the built-in ones ("tango", "pygments",
+#'  "kate", "monochrome", "espresso", "zenburn", "haddock" and "breezedark") or
+#'  among some custom theme passed as ".theme" files (see Details in the
+#'  \href{https://pandoc.org/MANUAL.html#syntax-highlighting}{Pandoc Manual}).
+#'  \pkg{rmarkdown} includes two custom themes to select with \code{highlight}
+#'  parameter:
+#'  \itemize{
+#'  \item{"arrow", an accessible color theme derived from \href{https://www.a11yproject.com/}{optimized for
```

Comment:
Not sure it's still true to say is derived from the a11y project?

Reply:
Yes I forgot to remove after having reworded following Alison PR in distill